src/HydraAuction/OnChain/StandingBid.hs

```diff
@@ -76,28 +75,30 @@ mkStandingBidValidator terms datum redeemer context =
     inOutsByAddress address =
       byAddress address $ txInInfoResolved <$> txInfoInputs info
     validNewBid :: StandingBidState -> StandingBidState -> Bool
-    validNewBid (StandingBidState oldApprovedBidders oldBid) (StandingBidState newApprovedBidders (Just newBidTerms)) =
-      traceIfFalse "Bidder not signed" (txSignedBy info (bidBidder newBidTerms))
-        && traceIfFalse
-          "Bidder is not approved"
-          (bidBidder newBidTerms `elem` bidders oldApprovedBidders)
-        && traceIfFalse
-          "Approved Bidders can not be modified"
-          (oldApprovedBidders == newApprovedBidders)
-        && case oldBid of
-          Just oldBidTerms ->
-            traceIfFalse "Bid increment is not greater than minimumBidIncrement" $
-              bidAmount oldBidTerms + minimumBidIncrement terms <= bidAmount newBidTerms
-          Nothing ->
-            traceIfFalse "Bid is not greater than startingBid" $
-              startingBid terms <= bidAmount newBidTerms
+    validNewBid (StandingBidState _oldApprovedBidders oldBid) (StandingBidState _newApprovedBidders (Just newBidTerms)) =
+      -- FIXME: disabled until M6
```

Comment:
Why do we need to remove this until m6? We should be able to make this work already

Reply:
This breaks, cuz delegate is signing Tx, while it is not in list of bidders. And knowing delegate list by Frontend CLI also requires new code.

And in M6 solution will be changed to completely different anyway.